lib/rex/proto/kerberos/README.md

```diff
@@ -0,0 +1,94 @@
+# Rex Kerberos Protocol
```

Comment:
This additional context is great, is there a way to make this more accessible to a developer?  I see a few other locations where a README.md was used to add context for segments of functionality however the other locations seem like instruction for end users vs developer/code documentation.  Is there some way to shift this to show up in `rdocs` or in the new wiki documentation?

Reply:
We'll have separate set of docs for end users, either specifically dedicated to Kerberos, or Active Directory as a whole.

I opted to keep most of this information in a dedicated readme, as I felt it was more discoverable to future developers than being placed inline at the top of a Ruby file. Happy to iterate on this again though.